### DISCUSSION TITLE
Inspect: CSV Download should interpolate panel title when generating file name

### DISCUSSION THREAD

**Discussion by gilkgit** [grafana/grafana#38398] [Dashboards, Panels & Visualizations]
I have a chart with repleted variables in the title (The title is "Regression in $datacenter - $errorName").
When I export to CSV, the file name looks like this
"Regression in $datacenter - $errorName-data-2020-07-10 12_21_00.csv"
(Variables are not interpolated)

**Comment by dprokop** [comment]
@gilkgit can you provide more information about Grafana version you use?

**Comment by gilkgit** [comment]
Sorry, it's on 7.0.4

**Comment by Hualin-AUG** [comment]
I think this is a reasonable feature to have. My project really needs this feature for the customer

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
